### ISSUE TITLE
Shipping with Apple Silicon

### ISSUE THREAD

**Issue by bismark64** [boltops-tools/ufo#182]
Is there any place in the vars or config to specify a specific docker image platform?

Using Apple silicon chips to build the image creates an incompatibility issue when shipping to AWS ECS. 

I had to manually create the image using: 
`UFO_DOCKER_BUILD_OPTIONS="--platform=linux/amd64" UFO_ENV=env ufo docker build --push`

then manually ship it:
`UFO_ENV=env ufo ship --no-docker`
but I also had to tweak the config files in order to specify the correct image and for some reason this last command did not create a load balancer, so I had to add `config.elb.enabled = true`. 

So in short, I found several issues when running the commands like that.

Is there a way to specify the platform (`--platform=linux/amd64`) when shipping using the standard `ufo ship` command?
